probe_scraper/scrapers/git_scraper.py

```diff
@@ -109,22 +109,23 @@ def get_commits(
     ref: str,
     only_ref: bool = False,
     deprecated: bool = False,
+    before: Optional[float] = None,
```

Comment:
> If I understand it right probe-scraper doesn't run on weekends, so with just `$TODAY` we would miss out on commits that happened on the weekend.

hmm, yeah this would definitely miss out on weekend commits, then. With airflow the `{{ ds }}` variable is the prior date, so when probe scrapers runs on monday morning UTC `{{ ds }}` is friday. Thankfully that makes this easy to address, i just need to remove this `before` variable, so that we scrape the history from `--glean-limit-date` through the present.